src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -2025,4 +2025,8 @@ Additional information: {1}</value>
   <data name="Prefer_collection_expression" xml:space="preserve">
     <value>Prefer collection expression</value>
   </data>
+  <data name="void_is_not_a_valid_type_for_a_parameter" xml:space="preserve">
+    <value>'void' is not a valid type for a parameter</value>
```

Comment:
The message can probably be improved

Reply:
how about 'System.Void' instead.  That way if the user writes out that type explicitly (in C# or VB) it also doesn't work.